bindings/gumjs/gumscript-runtime-objc.js

```diff
@@ -1031,6 +1089,9 @@
                 if (specifier.hasOwnProperty('subclasses'))
                     subclasses = specifier.subclasses;
             }
+            if (typeof cls === 'string') {
```

Comment:
Hmm, it just makes me wonder if we should also do the same for `registerClass()`; should you be able to use strings for `super` and `protocols`? In the end my gut feeling is that we should go explicit, as that will encourage users to re-use and get caching benefits of using the same object over and over:

```JavaScript
const NSString = ObjC.classes.NSString;

ObjC.choose(NSString, {...});

Interceptor.attach(NSString.foo.implementation, {...});
```